File: src/main/java/com/huang/pojo/Department.java
package com.huang.pojo;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 部门表
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Department {
    @TableId(type = IdType.AUTO)
    private Integer id;
    private String departmentName;
}

File: src/main/kotlin/com/huang/pojo/Department.kt
package com.huang.pojo

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import javax.validation.constraints.NotBlank

/**
 * 部门表
 */
data class Department(
    @TableId(type = IdType.AUTO)
    var id: Int?,

    @field:NotBlank(message = "部门名称不能为空！")
    var departmentName: String,
)